qtranslate.php

```diff
@@ -54,7 +54,7 @@
 	ja by Brian Parker
 	nl by RobV
 	pt by netolazaro
-	pt-br by Pedro Mendonça
+	pt-pt by Pedro Mendonça
```

Comment:
This line is not about translation. You added 'pt-br' to the default language list, providing all the option, and this is a valuable contribution :+1: . Credit as translator is below in the line "pt_PT by Pedro Mendonça, claudiotereso".